src/base/emu-i386/simx86/codegen-x86.c

```diff
@@ -3118,6 +3103,8 @@ static unsigned int CloseAndExec_x86(unsigned int PC, int mode, int ln)
 	e_markpage(G->seqbase, G->seqlen);
 	e_mprotect(G->seqbase, G->seqlen);
 	G->cs = LONG_CS;
+	/* check links INSIDE current node */
+	NodeLinker(G, G);
```

Comment:
I haven't looked into a linker code myself.
How difficult/does it makes sense to split
it into 2 functions, instead of calling with (G, G)?

Reply:
The only difference between G, G and LastNodeX, G is that it outputs different debug info and sets `G->flags|=F_SLFL` (which is never checked for in the code but can be handy for debugging).

So it doesn't really make sense to split.